### ISSUE TITLE
Cleanup deprecations in guidemaker-ember-template

### ISSUE THREAD

**Issue by jenweber** [ember-learn/guidemaker-ember-template#65]
## Description

This work should  be done on the branch `website-redesign`. We updated the Ember version, and so we can see new deprecations. The goal of this ticket is to resolve the deprecations. This may be done in several PRs.

To-do:
- Run the tests locally on your computer
- See which deprecation warnings show up
- Pick one to fix
- Comment below with the one you are working on
- Make sure someone else is not already doing that one by reading other people's comments

## Supporting information

You can see the progress so far at https://github.com/ember-learn/guidemaker-ember-template/pull/61

**Comment by sumitd94** [comment]
I will start checking the deprecation and start working on it. I Will comment here on what I am fixing 👍

**Comment by mansona** [comment]
hey @sumitd94 👋  sorry I didn't see you message before now, are you still hoping to work on this? Do you need any help?

**Comment by sumitd94** [comment]
> hey @sumitd94 👋 sorry I didn't see your message before now, are you still hoping to work on this? Do you need any help?

yes, I am still on it. I will reach out to you incase I need any help :) Thanks!

**Comment by mansona** [comment]
A lot of the deprecations are actually coming from Guidemaker itself right now.

We're currently blocked on fixing this (and getting CI green again) with https://github.com/empress/guidemaker-default-template/pull/21 which is waiting on a release of ember-power-select.

Once that is merged (and released) we can release a version of guidemaker that has a fully green CI (with no deprecations) and then we can finally fix the deprecations in guidemaker-ember-template 🎉

**Comment by MinThaMie** [comment]
We are quite up to date right now, so I'm closing this issue
